docs/products/postgresql/reference/idle-connections.rst

```diff
@@ -0,0 +1,26 @@
+Idle connections
+================
+
+How Aiven for PostgreSQL® handles idle database connection sessions and uses TCP keep-alive.
+
+Currently, the following default keep-alive timeouts are used on the server-side:
+
+-  ``tcp_keepalives_idle=7200``
+
+-  ``tcp_keepalive_count=9``
+
+-  ``tcp_keepalives_interval=75``
+
+You can set the client-side keep-alive timeouts to whatever values you
+want. For more information, see
+https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-KEEPALIVES
+(the ``libpq`` configuration keys are the same as on the server side
```

Comment:
I don't understand how this related to the sentence before, maybe some more info is needed

Reply:
reworded.